tractolearn/generative/filter_generated_points.py

```diff
@@ -0,0 +1,657 @@
+# -*- coding: utf-8 -*-
```

Comment:
I'd rename the module:
- A module should not contain an action verb, as opposed to a script that actually performs some processing.
- `points` is maybe a vague term.

This module contains methods that belong to some other module, maybe `analysis`.  So the `filter` term is also vague here. This, essentially, is performing the sample data discrimination.

Reply:
I moved the file in the filtering folder and called it: `streamline_space_filtering.py`

Reply:
Done